src/transport/SecureSessionTable.cpp

```diff
@@ -99,32 +105,78 @@ SecureSession * SecureSessionTable::EvictAndAllocate(uint16_t localSessionId, Se
 
     auto cleanup = MakeDefer([this]() { mRunningEvictionLogic = false; });
 
+#if CONFIG_BUILD_FOR_HOST_UNIT_TEST
+    const size_t kMaxSessionTableSize = mMaxSessionTableSize;
```

Comment:
Feels like this should be factored out into a method, since we have it in multiple places.